test/server/python_deps.jl

```diff
@@ -0,0 +1,32 @@
+# this script uses the default Conda settings to get python, pyopenssl, and pyftpdlib
+# if you already have an acceptable version of python and pip installed it will attempt to use those instead
+
+using Conda
+using Compat
+
+py_version(python::AbstractString) = chomp(readstring(`$python -c "import platform; print(platform.python_version())"`))
+const python = try
+    py = get(ENV, "PYTHON", isfile("PYTHON") ? readchomp("PYTHON") : "python")
+    version = convert(VersionNumber, py_version(py))
+    if version < v"2.7.11"
+        error("Python version $vers < 2.7.11 is not supported")
+    elseif version >= v"3.0"
+        error("Python version $vers >= 3 is not supported")
+    else
+        run(`pip install pyopenssl`)
+    end
+    py
+catch error
+    info( "No system-wide Python was found; got the following error:\n",
+          "$error\nusing the Python distribution in the Conda package")
+    Conda.add("pyopenssl")
+    abspath(Conda.PYTHONDIR, "python" * ( is_windows() ? ".exe" : ""))
+end
+
+pip = (dirname(python) == abspath(Conda.PYTHONDIR))? abspath(Conda.SCRIPTDIR, "pip" * ( is_windows() ? ".exe" : "")): "pip"
+run(`$pip install pyftpdlib`)
```

Comment:
I'll have to double check to be sure, but I believe the issue I was having was that pyftpdlib wasn't available through Conda, but pyopenssl was. 
Using Conda to install pyopenssl triggers it to install the latest python 2.7 and pip into the Conda deps directory as part of the installation of miniconda. This is what Pycall does as well. It is by no means the best solution, but it does install the right version of python into a predictable place on all platforms. It gets everything up and running until I can find a better solution